.github/workflows/build-bootc.yaml

```diff
@@ -95,3 +100,63 @@ jobs:
           cosign sign \
             --yes \
             ${{ env.REGISTRY }}/${{ env.REPOSITORY }}/flightctl-agent-${{ matrix.flavor }}@${{ steps.push.outputs.digest }}
+
+      - name: Install podman
+        run: sudo apt install podman
+
+      - name: Build and push disk images
+        run: |
+          mkdir output
+          
+          URL=${{ env.REGISTRY }}/${{ env.REPOSITORY}}/flightctl-agent-${{ matrix.flavor }}
+          
+          oras login -u "${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_USERNAME }}" -p "${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_PASSWORD }}" "${{ env.REGISTRY }}"
+
+          oras blob push $URL .github/workflow-scripts/empty.json
+          cp .github/workflow-scripts/index.json-head index.json
+
+          export SIZE
+          export SHA
+          export ARCH
+          export FORMAT
+
+          for ARCH in amd64 arm64; do
+          for FORMAT in qcow2 raw; do
+          
+            sudo podman run --rm \
+              -it \
+              --privileged \
+              --pull=newer \
+              --security-opt label=type:unconfined_t \
+              -v $(pwd)/output:/output \
+              -v /var/lib/containers/storage:/var/lib/containers/storage \
+              quay.io/centos-bootc/bootc-image-builder:latest \
+                --type $FORMAT \
+                --target-arch $ARCH \
+                "$URL:bootstrap"
+            
+            if [ -f output/image/disk.raw ]; then
+              ARTIFACT=output/image/disk.raw
+            else
+              ARTIFACT=output/qcow2/disk.qcow2
+            fi
+
+            oras blob push "$URL" $ARTIFACT
+            
+            SHA="sha256:"$(sha256sum $ARTIFACT | awk '{print $1}')
```

Comment:
Hrm why are we having to calculate SHAs manually here?

Reply:
Oras AFAIK has no built-in tools for creating image indexes (only for uploading them via `oras manifest push`), so I build the jsons by hand.

Reply:
Urg that's annoying, I think we could do it all with buildah 

```
Generate raw, qcow2
gzip raw, qcow2
buildah manifest create $IMAGE
buildah manifest add $IMAGE --artifact <gzipped raw> --arch arm64 --os linux --artifact-type application/vnd.diskimage+raw
buildah manifest add $IMAGE --artifact <gzipped qcow2> --arch arm64 --os linux --artifact-type application/vnd.diskimage+qcow2
buildah manifest annotate --index --annotation bootc-image=<source-image> --annotation bootc-digest=<source-digest> $IMAGE
buildah manifest push --all $IMAGE
```
I just don't think we need to / should be building the image index + child manifests manually :/

Reply:
Or even crate the manifest with buildah and then add to it with ORAS

Reply:
I was going to say that `buildah push` always produces an invalid manifest for me and that's why I went this route, but I realized that I had not been including the `--all` flag haha... Just pushed a new script that uses buildah.